README.md

```diff
@@ -152,3 +152,289 @@ commit it in this repo.
 ```
 just tf grafana apply
 ```
+## Updating the Deployment
+
+This deployment uses
+[cardano-parts](https://github.com/input-output-hk/cardano-parts), which
+periodically releases new features and `cardano-node` versions. Therefore,
+it’s necessary to update the `cardano-parts` version in this repository. This
+section provides a step-by-step guide to perform the update.
+
+### Environment Setup
+
+Before starting, ensure that you have the appropriate environment:
+
+- **Using the `dev-deployer` machine**: The `direnv` tool will automatically
+  load the Nix development shell.
+- **Using a local copy**: If working from a local repository, run the
+  following command to enable the necessary Nix features:
+
+  ```bash
+  nix --extra-experimental-features fetch-closure develop
+  ```
+  _Note_: Updating from the `dev-deployer` machine is recommended because you
+  can immediately test the changes. However, updating from a local copy makes
+  opening a PR easier, as the `dev-deployer` machine lacks GitHub credentials.
+
+### Step 1: Check Current `cardano-parts` Version
+
+To check the current version of `cardano-parts`, run:
+
+```bash
+nix flake metadata | grep cardano-parts
+```
+
+This command will return an output similar to the following:
+
+```
+├───cardano-parts: github:input-output-hk/cardano-parts/01f54d8feac449f846f33988f7d74711a1b856da
+│   │   ├───nixpkgs follows input 'cardano-parts/nixpkgs'
+│   │   ├───nixpkgs follows input 'cardano-parts/nixpkgs'
+│   │   │   └───nixpkgs follows input 'cardano-parts/haskell-nix/hydra/nix/nixpkgs'
+│   │   ├───nixpkgs follows input 'cardano-parts/haskell-nix/nixpkgs-unstable'
+│   │   └───stackage follows input 'cardano-parts/empty-flake'
+│       └───nixpkgs follows input 'cardano-parts/nixpkgs'
+├───flake-parts follows input 'cardano-parts/flake-parts'
+├───nixpkgs follows input 'cardano-parts/nixpkgs'
+└───nixpkgs-unstable follows input 'cardano-parts/nixpkgs-unstable'
+```
+
+At the time of writing, the repository uses the `cardano-parts` revision `01f54d8feac449f846f33988f7d74711a1b856da`.
+
+### Step 2: Identify the Associated PR
+
+To see which pull request (PR) corresponds to the current revision:
+
+1. Visit the [cardano-parts `main` commits page](https://github.com/input-output-hk/cardano-parts/commits/main).
+2. Locate the commit that matches your current revision, or go directly to the
+   [commit URL for the current version](https://github.com/input-output-hk/cardano-parts/commits/01f54d8feac449f846f33988f7d74711a1b856da).
+
+For example, revision `01f54d8feac449f846f33988f7d74711a1b856da` corresponds
+to [PR #48](https://github.com/input-output-hk/cardano-parts/pull/48).
+
```

Comment:
^^^ Sorry -- we'll see if we can make this pinning and comparison process a bit less painful in the near future.